examples/next/pages/ui/components/authenticator/sign-in-totp-sms-mfa/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Authenticator } from "aws-amplify-react";
```

Comment:
Should we add `@aws-amplify/ui-react` (and remove this dependency) to check for parity?